packages/app/src/app/account/[id]/transactions/page.tsx

```diff
@@ -11,7 +11,13 @@ export default async function AccountTransactions({
   params: { id },
 }: PageProps) {
   const txs = await getAccountTransactions({ owner: id });
-  return <TxList hidePagination transactions={txs.edges} />;
+  return (
+    <TxList
+      hidePagination
+      transactions={txs.edges}
+      className="mobile:p-0 tablet:p-0 desktop:p-0"
```

Comment:
`mobile` is not needed , we can use as default one

this applies also for other usages of `mobile:` in this PR

```suggestion
      className="p-0 tablet:p-0 desktop:p-0"
```